oraclepaas/resource_mysql_service_instance.go

```diff
@@ -72,12 +72,6 @@ func resourceOraclePAASMySQLServiceInstance() *schema.Resource {
 				}, true),
 			},
 
-			"enable_notification": {
-				Type:     schema.TypeBool,
-				Optional: true,
-				ForceNew: true,
-			},
```

Comment:
given this is defaulted below, I'm assuming we don't want to parse it out?

Reply:
If somebody is specifying a `NotificationEmail` than they have to set `EnableNotification` to `true`. You can't set `EnableNotification` without an email so we are just setting the value in terraform so the user doesn't have to